plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/TypeHelper.java

```diff
@@ -122,9 +122,10 @@ private static Type fromKuduClientType(org.apache.kudu.Type ktype, ColumnTypeAtt
                 return VarbinaryType.VARBINARY;
             case DECIMAL:
                 return DecimalType.createDecimalType(attributes.getPrecision(), attributes.getScale());
-            // TODO: add support for varchar and date types: https://github.com/trinodb/trino/issues/11009
-            case VARCHAR:
+            // TODO: add support for varchar types: https://github.com/trinodb/trino/issues/11009
```

Comment:
The comment place is wrong now. Please move to above `case VARCHAR:`

Reply:
Done